src/api/export.ts

```diff
@@ -186,7 +186,7 @@ async function _export({
 					const cleaned = clean_html(body);
 
 					const base_match = /<base ([\s\S]+?)>/m.exec(cleaned);
-					const base_href = base_match && get_href(base_match[1]);
+					const base_href = base_match ? get_href(base_match[1]) : '/';
```

Comment:
I think this would make it perfect:
```suggestion
					const base_href = base_match ? get_href(base_match[1]) : url.pathname;
```